common/guest/core/exception.c

```diff
@@ -1,18 +1,130 @@
+/* have to move */
+/* Access system register */
+#define read_sr64(name)       ({ unsigned long rval; asm volatile (\
+                              " mrs %0, "#name \
+                                : "=r" (rval) : : "memory", "cc"); rval; })
 
-#define ARCH_REGS_NUM_GPR   13
+#define read_sr32(name)       ({ unsigned int rval; asm volatile (\
+                              " mrs %0, "#name \
+                                : "=r" (rval) : : "memory", "cc"); rval; })
+
+#define write_sr64(name)      asm volatile (\
```

Comment:
Oh you're right. but it was not used. that is why this is works. Thanks, and it will be changed.